code/game/atoms.dm

```diff
@@ -2178,6 +2178,12 @@
 				//We inline a MAPTEXT() here, because there's no good way to statically add to a string like this
 				active_hud.screentip_text.maptext = "<span class='maptext' style='text-align: center; font-size: 32px; color: [active_hud.screentip_color]'>[name][extra_context]</span>"
 
+	//PARIAH EDIT ADDITION
```

Comment:
Unsure how to review properly, but remove cursor following during combat mode. This is more a preference on style and really I'm not of how it looks from what I've seen on servers with this.